docs/guides/define_agent.md

```diff
@@ -46,6 +46,15 @@ If you have [populated the local registry](./set_up.md#populate-the-local-regist
 
       * A reference to the {{fsm_app}} skill.
       * References to other components required by the agent (or dependencies of the {{fsm_app}} skill), under the relevant sections.
+
+        !!! warning "Important"
+
+            There are a number of components which are mandatory for agents that are part of a service:
+
+            * Connections: `valory/abci`, `valory/ipfs`.
+            * Protocols: `open_aea/signing`, `valory/abci`, `valory/ipfs`.
```

Comment:
A service can operate without IPFS connection and protocol. Instead, I suggest adding the ACN since it is necessary for the registration startup round and recovery.